<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>easing_preview_base</class>
 <widget class="QWidget" name="easing_preview_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="enable_easing">
     <property name="text">
      <string>Enable Easing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="easing_combo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>The pan easing will determine the motion 
characteristics of the camera on the Y axis 
as it flies across the scene.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="easing_preview" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>150</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enable_easing</sender>
   <signal>toggled(bool)</signal>
   <receiver>easing_combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
